src/probnum/diffeq/ode/ivp_examples.py

```diff
@@ -0,0 +1,464 @@
+# pylint: disable=unused-variable
```

Comment:
why this? Is it because most ODEs are autonomous, i.e. ignore t?

Reply:
I think this can be removed. At least pylint does not complain if I do so. Fixed